lib/Migration/Version1100Date20210304143008.php

```diff
@@ -78,13 +78,6 @@ public function changeSchema(IOutput $output, Closure $schemaClosure, array $opt
 				'length' => 4,
 			]);
 			$tagsMessageTable->setPrimaryKey(['id']);
-			$tagsMessageTable->addUniqueIndex(
```

Comment:
unrelated change, right?

Reply:
that should have been resolved by the rebase. I'm sorry - shall I leave it? It's in the rework tagging branch as well

Reply:
right, that was also in https://github.com/nextcloud/mail/pull/4840. so actually this PR doesn't drop anything anymore as the code is already gone. rebase after your fixup and it should be all good :)